src/puptoo/process/profile.py

```diff
@@ -631,6 +635,18 @@ def _get_gpg_pubkey_packages(rpms):
     return rpms.packages.get("gpg-pubkey", [])
 
 
+def _get_mssql_server_package(packages):
+    """
+    Get the mssql-server package from the latest packages from the
+    InstalledRpms parser.
+    """
+    result = None
+    for package in packages:
+        if package.name == "mssql-server":
+            result = package
+    return package
```

Comment:
`s/package/result/`

Reply:
thanks! this is what happens when I code late on a friday evidently :)